manager/allocator/networkallocator/networkallocator.go

```diff
@@ -657,7 +658,12 @@ func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, str
 }
 
 func (na *NetworkAllocator) loadDriver(name string) error {
-	_, err := plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	var err error
+	if pg := na.drvRegistry.GetPluginGetter(); pg != nil {
+		_, err = pg.Get(name, driverapi.NetworkPluginEndpointType, plugingetter.Lookup)
+	} else {
+		_, err = plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	}
```

Comment:
Can you also update the other instances of this pattern in libnetwork? Eg. https://github.com/docker/libnetwork/blob/4da563bb06cc7e488047ddf3a20cde2f9d9928cc/controller.go#L1089